automatic/kubernetes-kustomize/README.md

```diff
@@ -0,0 +1,18 @@
+# <img src="https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/master/icons/kubernetes-kustomize.png" width="48" height="48"/> [kubernetes-kustomize](https://chocolatey.org/packages/kubernetes-kustomize)
+
+## Kustomize
```

Comment:
This should be in "standard" Readme form

See https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#123-informative-description